tests/sanity/tests/collaborative/issues.spec.ts

```diff
@@ -108,4 +105,49 @@ test.describe('Collaborative test for issue', () => {
       status: 'In Progress'
     })
   })
+
+  test('First user change assignee, second user should see assigned issue', async ({ page, browser }) => {
+    const newAssignee: string = 'Dirak Kainin'
+    const issue: NewIssue = {
+      title: 'First user change assignee, second user should see assigned issue',
+      description: 'Issue for collaborative test'
+    }
+
+    // open second page
+    const userSecondPage = await getSecondPage(browser)
+    await (await userSecondPage.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+
+    await test.step('change assignee by first user', async () => {
+      await (await page.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+      const issuesPage = new IssuesPage(page)
+      await issuesPage.linkSidebarAll.click()
+      await issuesPage.modelSelectorBacklog.click()
+      await issuesPage.searchIssueByName(issue.title)
+      await issuesPage.openIssueByName(issue.title)
+
+      const issuesDetailsPage = new IssuesDetailsPage(page)
+      await issuesDetailsPage.editIssue({ assignee: newAssignee })
+    })
+
+    await test.step('change assignee by first user and check issue', async () => {
```

Comment:
now it is better. still couple of minor issues, though:
1. step description seems to be inaccurate ("change assignee by **first user**")
2. `and` in the step description indicates that step can be split into two separate steps "check notification" and "check issue"

Reply:
great point. =)
updated